eng/versioning/external_dependencies.txt

```diff
@@ -66,6 +66,7 @@ org.json:json;20140107
 org.linguafranca.pwdb:KeePassJava2;2.1.4
 org.powermock:powermock-api-mockito2;2.0.2
 org.powermock:powermock-module-junit4;2.0.2
+org.slf4j:jcl-over-slf4j;1.7.28
```

Comment:
Is bringing this logging bridge necessary?

/cc @JonathanGiles

Reply:
I'm not sure this is required - @chenrujun could you clarify why this is needed?

Reply:
I just copy it from original repo. 
I delete it then run `mvn clean install`, there is no error.
So I deleted it.